bears/hypertext/HTMLHintBear.py

```diff
@@ -0,0 +1,100 @@
+from coalib.bearlib.abstractions.Linter import linter
+from dependency_management.requirements.NpmRequirement import NpmRequirement
+
+
+@linter(
+    executable='htmlhint',
+    output_format='regex',
+    output_regex=r'(?P<filename>.+):(?P<line>\d+):(?P<column>\d+):\s*'
+                 r'(?P<message>.+)\s*(\d+)(\sproblems)\s*',
```

Comment:
Don't use capture groups other then ones coala uses.. the digit one and the `(\sproblems)`, btw is this regex correct? `\sproblems` will match `\nproblems`, `\tproblems`, etc. I don't think that is intended ?